src/components/select/template.njk

```diff
@@ -0,0 +1,18 @@
+{% from "label/macro.njk" import govukLabel %}
+{% set labelArguments = {
+  text: params.label.text if params.label.text else params.label,
+  html: params.label.html,
+  hintText: params.hintText if params.hintText else params.hintHtml,
+  for: params.id,
+  errorMessage: params.errorMessage
+} %}
+{%- if params.label %}
+{{- govukLabel(labelArguments) -}}
+{% endif -%}
+
+<select class="govuk-c-select
+  {%- if params.classes %} {{ params.classes }}{% endif %}{%- if params.errorMessage %} govuk-c-input--error{% endif %}" id="{{ params.id }}" name="{{ params.name }}{% for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}">
```

Comment:
Need a closing quote for the `name` attribute.

Reply:
Would also be good to have consistency around whitespace inside `{{` - `{{ params.id }}` vs `{{attribute}}`.